components/HackathonCard.js

```diff
@@ -0,0 +1,81 @@
+import styled from "styled-components"
+import Button from "./Button"
+
+const CardContainer = styled.div`
+    width: 380px;
+    height: 500px;
+    overflow: hidden;
+    border-radius: 12px;
+    margin: 2em;
+    position: relative;
+`
+
+const BackgroundImageContainer = styled.div`
+    background: linear-gradient( rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0.3) ), url(https://i.pinimg.com/474x/00/5e/95/005e953027d76c35ee6ec1446d43a739.jpg) center;
```

Comment:
Can we pull this out into a prop for this PR? Nice image choice 👀

Reply:
oh good catch. I wanted to do that but I forgot LOL. I will also put the currently hard-coded colors into the newly-merged ThemeProvider

Reply:
Sweet, thanks! Everything else looks great 🔥